docs/dns-stack.md

```diff
@@ -62,6 +62,12 @@ other queries are forwardet to the nameservers found in ``upstream_dns_servers``
 This does not install the dnsmasq DaemonSet and instructs kubelet to directly use kubedns/skydns for
 all queries.
 
+#### manual
+This does not install dnsmasq or kubedns, but allows you to specify
+`manual_dns_server`, which will be configured on nodes for handling Pod DNS.
```

Comment:
maybe an `external_dns_server` would fit better?

Reply:
That name seems more like upstream_dns_server, which is inaccurate. This is a server that performs kubedns services, but isn't managed by kubespray. This would allow someone to use a helm chart instead for cluster DNS